draft-ietf-quic-http.md

```diff
@@ -1814,6 +1814,15 @@ considerations about exhaustion of stream identifier space apply, though the
 space is significantly larger such that it is likely that other limits in QUIC
 are reached first, such as the limit on the connection flow control window.
 
+QUIC manages stream concurrency on behalf of HTTP/3.  QUIC considers a stream
+closed when all sent data and all acknowledgements for received data have been
+acknowledged by the peer.  HTTP/2 considers a stream closed when the frame
```

Comment:
I think this sentence is incorrect.

I think it should be: "QUIC considers a stream closed when all data have been received and all the sent data have been acknowledged."

Reply:
Or a reset has been sent and acknowledged.